akka-http2-support/src/main/scala/akka/http/scaladsl/Http2.scala

```diff
@@ -35,6 +35,49 @@ class Http2Ext(private val config: Config)(implicit val system: ActorSystem)
 
   val http = Http(system)
 
+  /**
+   * Handle requests using HTTP/2 immediately, without any TLS or negotiation layer.
+   */
+  def bindAndHandleRaw(
```

Comment:
Raw sounds a bit weird; `bindAndHandlePlainText`?  AFAIR that's also wording used in GRPC

Reply:
That's indeed what `io.grpc.netty` uses, but I found `PlainText` a bit weird since even though there's no encryption, it's still a binary protocol. 

This approach is described in the spec as [starting 'with prior knowledge'](http://httpwg.org/specs/rfc7540.html#known-http), but `bindAndHandleWithPriorKnowledge` is weird because it's the client that has prior knowledge, not the server.

Reply:
would it be possible to make it a configuration property or a special `connectionContext` instead?

I'd image that you might want to easily switch between this and the real thing, perhaps running (some) tests with this and full integration tests and production with the real thing.

Reply:
> That's indeed what io.grpc.netty uses, but I found PlainText a bit weird since even though there's no encryption, it's still a binary protocol.

Yeah, I've heard this argument but the general feel it conveys seems right "though shalt not use paintext in prod" ;-) Crypto wise I think plaintext is referred to non encrypted, regardless if binary or not?

Reply:
Ah, I meant this comment; Yeah trying to achieve this with a connection context would be preferable indeed, could you give it a look @raboof ?

Reply:
Basically it's the same as we have in Akka HTTP/1, where we have the plain and https one; the same could work here basically, even if we strongly recommend the https one